src/App.js

```diff
@@ -38,23 +40,34 @@ const App = () => {
     const names = objectsMapped.map(object => object.title)
     const artist = objectsMapped[0].artistDisplayName
     const bio = objectsMapped[0].artistDisplayBio
-    setImages(paintings)
-    setTitles(names)
-    setArtistName(artist)
-    setArtistBio(bio)
-     console.log (images)
+     setImages(paintings)
```

Comment:
In these function where you're using async await, because there is no `catch` block there is no way to handle errors. Look at `try/catch` blocks used with async await to be able to handle potential errors.